kodilla-patterns/src/main/java/com/kodilla/patterns/singleton/Logger.java

```diff
@@ -0,0 +1,26 @@
+package com.kodilla.patterns.singleton;
+
+public class Logger {
+
+    private String lastLog = "";
+    private static Logger loggerInstance = null;
+
+    private Logger() {
+    }
+
+    public static Logger getInstance() {
+        if (loggerInstance == null) {
```

Comment:
this is not thread-safe implementation. See http://www.vogella.com/tutorials/DesignPatternSingleton/article.html#singleton
or https://en.wikipedia.org/wiki/Singleton_pattern
There are many options how to do this, you can switch to enum (lazy initialization is not used in this case), or add proper synchronization to your method.